cloudevents-sdk-reqwest/src/client_request.rs

```diff
@@ -7,6 +7,17 @@ use cloudevents::Event;
 use reqwest::RequestBuilder;
 use std::str::FromStr;
 
+/// Extention Trait for [`RequestBuilderExt`]
+pub trait RequestBuilderExt {
```

Comment:
Can you move this and the impl too near the `event_to_request`?

Reply:
cool, I'll do the same for the other one too